docs/reference/modules/discovery/azure.asciidoc

```diff
@@ -1,6 +1,4 @@
 [[modules-discovery-azure]]
 === Azure Discovery
 
-Azure discovery allows to use the Azure APIs to perform automatic discovery (similar to multicast).
-Please check the https://github.com/elasticsearch/elasticsearch-cloud-azure[plugin website]
-to find the full documentation.
+Azure discovery is available as a plugin. See {plugins}/discovery-azure.html[discovery-azure] for more information.
```

Comment:
why the removal of the old single line description? I think that is better than just saying "there is a plugin"